descwl_shear_sims/shifts.py

```diff
@@ -66,12 +66,79 @@ def get_shifts(
                 buff=buff,
                 size=nobj,
             )
+        elif layout == 'hex':
+            shifts = get_hex_shifts(
+                rng=rng,
+                dim=coadd_dim,
+                buff=buff,
+                n_on_side=GRID_N_ON_SIDE,
```

Comment:
The stack is finding these to be blended at this density, but I think we want them to be not deblended by default.

Reply:
using `n_on_side` 5 instead of 6 seems to work when the objects are s/n = 50

however for the default higher s/n the entire thing gets called one footprint.  So I think this needs some tuning

Reply:
Does it work ok for 4?

Reply:
for 4 it still puts everything in one big footprint

Reply:
Wow really. Idk what to do then. This really is a bug in the stack code.